docs/components/datetimepicker.md

```diff
@@ -0,0 +1,409 @@
+# DateTime Picker
+
+<div v-if="hasMounted">
+
+**KDateTimePicker** - A date and time selection tool, displayed inside a popover.
+
+:::danger Experimental Component
+`KDateTimePicker` is an experimental component. The component functionality, props, and events may change until it is in stable status.
+:::
+
+## Examples
+
+### Single date
+
+Create a single date selection calendar
+Set the `v-model` to [Single date time picker](#single-date-time-picker-v-model)
+
+<div>
+  <KDateTimePicker
+    v-model="currentValue1"
+    @change="newVal => emitVal1 = newVal"
+    placeholder="Please select a date"
+    mode="date"
+    :range="false"
+  />
+  <div class="mt-6">Emitted value: <pre class="json">{{ JSON.stringify(emitVal1) }}</pre></div>
+</div>
+
+```html
+<KDateTimePicker
+  v-model="currentValue"
+  @change="newVal => emitVal = newVal"
+  placeholder="Please select a date"
+  mode="date"
+  :range="false"
+/>
+```
+
+<!-- ### Single time
+
+<div>
+  <KDateTimePicker
+    @change="newVal => emitVal2 = newVal"
+    placeholder="Please select a time"
+    mode="time"
+    :range="false"
+  />
+  <div class="mt-6">Emitted value: <pre class="json">{{ currentValue2 }}</pre></div>
+</div>
+
+```html
+<KDateTimePicker
+  @change="newVal => emitVal = newVal"
+  placeholder="Please select a time"
+  mode="time"
+  :range="false"
+/>
+``` -->
+
+### Single date and time
+
+Set the `v-model` to [Single date time picker](#single-date-time-picker-v-model)
+
+<div>
+  <KDateTimePicker
+    v-model="currentValue3"
+    @change="newVal => emitVal3 = newVal"
+    placeholder="Please select a date and time"
+    mode="dateTime"
+    :range="false"
+  />
+  <div class="mt-6">Emitted value: <pre class="json">{{ JSON.stringify(emitVal3) }}</pre></div>
+</div>
+
+```html
+<KDateTimePicker
+  v-model="currentValue"
+  @change="newVal => emitVal = newVal"
+  placeholder="Please select a date and time"
+  mode="dateTime"
+  :range="false"
+/>
+```
+
+### Date and time range
+
+Set the `v-model` to [Range date time picker](#range-date-time-picker-v-model)
+
+<div>
+  <KDateTimePicker
+    v-model="currentValue4"
+    @change="newVal => emitVal4 = newVal"
+    placeholder="Please select a date and time"
+    mode="dateTime"
+    :range="true"
+  />
+  <div class="mt-6">Emitted value: <pre class="json">{{ emitVal4 }}</pre></div>
+</div>
+
+```html
+<KDateTimePicker
+  v-model="currentValue"
+  @change="newVal => emitVal = newVal"
+  placeholder="Please select a date and time"
+  mode="dateTime"
+  :range="true"
+/>
+```
+
+### Calendar and Relative time frames
+
+Set the `v-model` to [Range date time picker](#range-date-time-picker-v-model)
+
+Display both a calendar and relative time frames, passing in a preset week-long range.
+This instance also makes use of the `minDate` and `maxDate` parameters, which are both optional.
```

Comment:
Since you're demoing how to use `minDate` and `maxDate`, can you include the values you are setting these to in the example code that is displayed instead of just the variable?

```
:minDate="minDate"
:maxDate="maxDate"
```